.cirrus.yml

```diff
@@ -0,0 +1,57 @@
+task:
+  auto_cancellation: true
+
+  env:
+    matrix:
+      PY_VER: "3.6"
+      PY_VER: "3.7"
+
+  matrix:
+    - name: linux
+      container:
+        image: gcc:latest
+      env:
+        PATH: $HOME/conda/bin:$PATH
+      system_script:
+        - apt-get update
+        - apt-get install -y libgl1-mesa-glx
```

Comment:
This looks ominous... What's this doing and could it be related to the crash?

Reply:
This was put into place to fix an issue in linux where OpenGL did not have the proper backend installed on the system. I additionally investigated using the `nvidia/opengl:base` Docker image (and other images) but ran into the problem where somehow `curl` was unavailable as a command so I added these lines instead.